docs/basics/getting-started.md

```diff
@@ -1,210 +1,159 @@
-
-:::tip
-There are many other [config options available](./options.md#all-options) that
-you should thoroughly look through to achieve your desired results.
-:::
-
-Once you've finished with your config file, the following command can get you
-started running a full scan of your torrent directory!
+Or, if you are using Docker, make sure to expose the paths cross-seed needs to
+access:
 
 ```shell
-cross-seed search
-```
-
-### with Docker
-
-#### Set up your container
-
-With Docker, any paths that you would provide as command-line arguments or in
-the config file should stay hardcoded. Instead, you can mount volumes to the
-Docker container.
-
--   `/config` - config dir (this follows a common Docker convention)
--   `/torrents` - your torrent input dir
-    -   usually set to your rTorrent session dir, Deluge state dir, Transmission torrents dir, or your qBittorrent BT_backup dir
--   `/cross-seeds` - the output dir. People sometimes point this at a watch folder. If you use
-
-Create or open your existing `docker-compose.yml` file and add the `cross-seed`
-service:
-
-```yaml
-version: "2.1"
-services:
-    cross-seed:
-        image: ghcr.io/cross-seed/cross-seed
-        container_name: cross-seed
-        user: 1000:1000 # optional but recommended
-        ports:
-            - "2468:2468" # you'll need this for daemon mode only
-        volumes:
-            - /path/to/config/folder:/config
-            - /path/to/torrent_dir:/torrents:ro # your torrent clients .torrent cache, can and should be mounted read-only (e.g. qbit: `BT_Backup` | deluge: `state` | transmission: `transmission/torrents` | rtorrent: session dir from `.rtorrent.rc`)
-            - /path/to/output/folder:/cross-seeds
-            - /path/to/torrent/data:/data # OPTIONAL!!! this is location of your data (used for data-based searches or linking)
-              # will need to mirror your torrent client's path (like Arr's do)
-        command: daemon # this enables the daemon, change to search to specifically run a search ONLY
-        restart: "no" # make sure to not enable it in search mode, otherwise it will keep rerunning constantly
+docker run \
+-v /path/to/config:/config \
+-v /path/to/torrent_dir:/path/to/torrent_dir \
+-v /path/to/output:/path/to/output \
+ghcr.io/cross-seed/cross-seed daemon
 ```
 
-#### Create a config file
-
-When you run the container the first time, it will create a config file at
-`/config/config.js`. Open this file in your favorite editor. Since you've
-already configured your volume mappings, the only required config option is
-[`torznab`](../basics/options.md#torznab).
+After a few moments, you should see `cross-seed` automatically starting to
+search for things in your catalog. It will also automatically scan your
+trackers' RSS feeds for new releases you can cross-seed.
 
-:::tip
-There are many other [config options available](./options.md#all-options) that
-you should thoroughly look through to achieve your desired results.
-:::
```

Comment:
Haven't even mentioned `searchCadence` at this point. If this assumes default, perhaps explaining the existence of the cadence options in some form would be beneficial, even if "in passing" and linking to their options page. In my mind, that would at least give the inclination that this is not all hard-coded and preset, and would be something I can look into being configurable.

The reasoning being, I often see people go through the 5 or so options that are "required" and never look further before running things and expecting psychic abilities. Then pop up in discord asking me how to do things that would otherwise be obvious had it been mentioned that these options existed more "in-their-face" and needed review or attention if they weren't to their liking.

I understand this might take some thought on how to do it without overwhelming the user, but given the propensity to "do what the guide says and nothing more" rather than "review the docs and see what cross-seed can do before running" - it might be beneficial to just at least drop a note about the cadence's existence somewhere around here, if nothing else.

Reply:
Regarding the "general" case, I think I want to break out some of these common use cases I think you're alluding to into their own pages in the Tutorials section. Things like 
- linking
- partial matching (maybe combined with linking?)
- usenet 
- autobrr
- 

As for searchCadence specifically, I am hoping the defaults will be satisfactory, but I think you're right that it makes sense to at least mention the configurability.